addons/base_setup/models/ir_http.py

```diff
@@ -9,5 +9,6 @@ class IrHttp(models.AbstractModel):
 
     def session_info(self):
         result = super(IrHttp, self).session_info()
-        result['show_effect'] = request.env['ir.config_parameter'].sudo().get_param('base_setup.show_effect')
+        if request.env.user.has_group('base.group_user'):
```

Comment:
One small question though: sometimes we test the group membership for `self.env.user` and sometimes `request.env.user`. Those will most often be aligned, but it's not guaranteed.
Shouldn't we be consistent everywhere? Or did we discuss this and I forgot already? ;-)

Reply:
we did not discuss it

IIRC I wanted to use the `request` when it was already imported (since in that case most of the code used it to access i.e. the environment and i wanted to be consistent with the existing code), otherwise `self` (since I did not want to add an import). It seems I did not manage to be entirely consistent with that arbitrary rule...